Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM mcr.microsoft.com/vscode/devcontainers/base:0.202.7-bullseye
+ARG USER=vscode
+ARG UID=1000
+ARG GID=${UID}
+ARG NIX_INSTALLER=https://releases.nixos.org/nix/nix-2.10.3/install
+ARG CHANNEL_URL=https://github.com/NixOS/nixpkgs/archive/aaa1c973c8c189195e1b1a702d3b74dbcde91538.tar.gz
+ARG CACHIX_NAME=composable-community
+
+SHELL [ "/bin/bash", "-o", "pipefail", "-o", "errexit", "-c" ]
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get install --yes --no-install-recommends \
+    ca-certificates \
+    curl \
+    sudo \
+    xz-utils    
```

Comment:
```suggestion
ENV DEBIAN_FRONTEND=noninteractive

RUN apt-get update && \
    apt-get install --yes --no-install-recommends \
    ca-certificates \
    curl \
    sudo \
    xz-utils    
```

Reply:
ah, it is non local. assumption that debian will do wrf only in its scope of run. we do. ot expect wrf out of scope cause just basic unix and nix right after

Reply:
so we no tell there was other case then apt warning

Reply:
merged into one run